etc/kayobe/containers/pulp/post.yml

```diff
@@ -27,3 +27,10 @@
   when:
     - stackhpc_pulp_sync_for_local_container_build | bool
     - pulp_settings.changed
+
+- name: Login to docker registry
```

Comment:
We do have `etc/kayobe/ansible/docker-registry-login.yml. Would it be better to call that?

Reply:
That's currently only in the trivy image scanning branch: https://github.com/stackhpc/stackhpc-kayobe-config/blob/feat/trivy-image-scanning-test/etc/kayobe/ansible/docker-registry-login.yml

Reply:
:doughnut: